frontends/main/src/app/c/[channelType]/[name]/page.tsx

```diff
@@ -25,7 +25,6 @@ export async function generateMetadata({
     searchParams,
     title: `${channelDetails.title}`,
     description: channelDetails.public_description,
-    image: channelDetails.configuration.logo,
```

Comment:
This is unrelated to the other changes, but...
- we noticed the metadataBase issue because channel og:image were not working
- `og:image` cannot be an SVG, so without this change, they still won't work.